docker-compose.yml

```diff
@@ -100,6 +100,7 @@ services:
       - GeneralSettings:LanguageFilesLocation=Languages/
       - TestdataRepositorySettings:RepositoryLocation=/app/Testdata
       - TestdataRepositorySettings:DesignerHost=altinn-designer
+      - PlatformSettings:ApiAuthenticationEndpoint=http://localhost:5040/authentication/api/v1/
```

Comment:
Do we need this variable both here and in appsettings or would one of the places do?

Reply:
yes, we need at both places, when deployed in containers, the value from this environment variable is used and the one in appsettings is used when its deployed locally in our machine